test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTest.java

```diff
@@ -368,6 +368,22 @@ public void periodicCkpointTrimTest() throws Exception {
         }
     }
 
+    /** We have some non-determinism that needs more runtime to
+     *  try to find buggy executions.  These tests (and the wall
+     *  clock time they consume on each test run) can go away
+     *  sometime, after "soak time" and another round of review.
+     */
+    @Test public void periodicCkpointTrimTest0() throws Exception { periodicCkpointTrimTest(); }
```

Comment:
@Maithem The simplest loop would hide how frequently the test fails. This way gives more verbose reporting. {shrug} This is intended to be temporary, until I finish alternative test technique to force different thread execution schedules.